src/collections/Form/FormFields.js

```diff
@@ -17,22 +17,25 @@ export default class Fields extends Component {
 
   static _meta = {
     library: META.library.semanticUI,
-    name: 'Fields',
+    name: 'FormFields',
     parent: 'Form',
     type: META.type.collection,
   };
 
   render() {
     let fieldCount = 0
-    Children.forEach(this.props.children, child => {
-      if (_.get(child, 'type._meta.name') === 'Field') fieldCount += 1
-    })
-    fieldCount = numberToWord(fieldCount)
+    if (this.props.evenlyDivided) {
+      Children.forEach(this.props.children, child => {
+        if (_.get(child, 'type._meta.name') === 'FormField') {
+          fieldCount += 1
+        }
+      })
+    }
```

Comment:
Only count form field children if we are `evenlyDivided` so we are wasting cycles every render for no reason.